stable/prometheus/templates/pushgateway-deployment.yaml

```diff
@@ -38,11 +38,7 @@ spec:
             - containerPort: 9091
           readinessProbe:
             httpGet:
-            {{- if (index .Values "pushgateway" "extraArgs" "web.route-prefix") }}
-              path: /{{ index .Values "pushgateway" "extraArgs" "web.route-prefix" }}/#/status
-            {{- else }}
-              path: /#/status
-            {{- end }}
+              path: /-/ready
```

Comment:
Doesn't web.route-prefix need to be taken into consideration?

Reply:
indeed.